docs/src/content/docs/de/components/badges.mdx

```diff
@@ -0,0 +1,150 @@
+
+Standardmäßig wird für das Abzeichen die Akzentfarbe des Aussehens deiner Website verwendet.
+Um eine eingebaute Abzeichen-Farbe zu verwenden, setze das Attribut [`variant`](#variant) auf einen der unterstützten Werte.
+
+<Preview>
+
+```mdx
+import { Badge } from '@astrojs/starlight/components';
+
+<Badge text="Hinweis" variant="note" />
+<Badge text="Erfolg" variant="success" />
+<Badge text="Tipp" variant="tip" />
+<Badge text="Warnung" variant="caution" />
+<Badge text="Gefahr" variant="danger" />
+```
+
+<Fragment slot="markdoc">
+
+```markdoc
+{% badge text="Hinweis" variant="note" /%}
+{% badge text="Erfolg" variant="success" /%}
+{% badge text="Tipp" variant="tip" /%}
+{% badge text="Warnung" variant="caution" /%}
+{% badge text="Gefahr" variant="danger" /%}
+```
+
+</Fragment>
+
+<Fragment slot="preview">
+	<Badge text="Hinweis" variant="note" />
+	<Badge text="Erfolg" variant="success" />
+	<Badge text="Tipp" variant="tip" />
+	<Badge text="Warnung" variant="caution" />
+	<Badge text="Gefahr" variant="danger" />
+</Fragment>
+
+</Preview>
+
+### Verschiedene Größen verwenden
+
+Verwende das Attribut [`size`](#size), um die Größe des Abzeichentextes zu steuern.
+
+<Preview>
+
+```mdx /size="\w+"/
+import { Badge } from '@astrojs/starlight/components';
+
+<Badge text="Neu" size="small" />
+<Badge text="Neu und verbessert" size="medium" />
+<Badge text="Neu, verbessert und größer" size="large" />
+```
+
+<Fragment slot="markdoc">
+
+```markdoc /size="\w+"/
+{% badge text="Neu" size="small" /%}
+{% badge text="Neu und verbessert" size="medium" /%}
+{% badge text="Neu, verbessert und größer" size="large" /%}
+```
+
+</Fragment>
+
+<Fragment slot="preview">
+	<Badge text="Neu" size="small" />
+	<Badge text="Neu und verbessert" size="medium" />
+	<Badge text="Neu, verbessert und größer" size="large" />
+</Fragment>
+
+</Preview>
+
+### Abzeichen anpassen
+
+Passe Abzeichen an, indem du andere `<span>`-Attribute wie `class` oder `style` mit benutzerdefiniertem CSS verwendest.
+
+<Preview>
+
+```mdx "style={{ fontStyle: 'italic' }}"
+import { Badge } from '@astrojs/starlight/components';
+
+<Badge text="Individuell" variant="success" style={{ fontStyle: 'italic' }} />
+```
+
+<Fragment slot="markdoc">
+
+```markdoc 'style="font-style: italic;"'
+{% badge text="Individuell" variant="success" style="font-style: italic;" /%}
+```
+
+</Fragment>
+
+<Badge
+	slot="preview"
+	text="Individuell"
+	variant="success"
+	style={{ fontStyle: 'italic' }}
+/>
+
+</Preview>
+
+## `<Badge>` Eigenschaften
+
+**Implementation:** [`Badge.astro`](https://github.com/withastro/starlight/blob/main/packages/starlight/user-components/Badge.astro)
+
+Die `<Badge>`-Komponente akzeptiert die folgenden Eigenschaften und auch alle [anderen `<span>`-Attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes):
+
+### `text`
+
+**erforderlich**  
```

Comment:
```suggestion
**Erforderlich**  
```

Should this be capitalized for consistency with "Typ" and "Voreinstellung"? Let me know what you think.

Reply:
imho it should not be capitalized...

why?

required (and erforderlich as well) are adjectives 

type (and Typ and Voreinstellung) are nouns

in German, the rules for capitalization are complex but generally speaking only nouns are capitalized.

someone could argue that all three must be written capitalized not only because of consistency but also because it's the first letter of a new "sentence" / "paragraph". but I think it's not a sentence and therefore it can be written small...

same as before: would love to hear some comments from other native speakers...

Reply:
Hmm.. Personally, I would prefer to capitalize all of them just for the sake of consistency. Even though 'erforderlich' is an adjective, I believe that capitalizing it along with 'Typ' and 'Voreinstellung' looks cleaner and more uniform. From a design perspective, this approach seems more visually balanced.